### ISSUE TITLE
Static image import does not work with webpack 4

### ISSUE THREAD

**Issue by QuentinRoy** [vercel/next.js#27976]
### What version of Next.js are you using?

11.1.0

### What version of Node.js are you using?

14.0.0

### What browser are you using?

any

### What operating system are you using?

Mac OS

### How are you deploying your application?

Vercel

### Describe the Bug

Importing image statically breaks compilation when using `webpack5: false`.

### Expected Behavior

The compilation shouldn't be broken, or the limitation should be reported in the documentation.

### To Reproduce

```js
import profilePic from "public/images/me.jpg";

export default function Home(){
  return <Image src={profilePic} />;
}
```

**Comment by QuentinRoy** [comment]
I suppose you meant I cannot?
I cannot yet update to webpack 5 because of #24700.

**Comment by ijjk** [comment]
Closing as this is expected to only work with webpack 5 and will be documented in https://github.com/vercel/next.js/pull/29186

Also please let us know if https://github.com/vercel/next.js/issues/24700#issuecomment-905545825 unblocks you from upgrading to webpack 5 or not so we can ensure that issue is fully resolved!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
